website/src/components/MacroComponents/MacrobenchMobile/MacrobenchMobile.jsx

```diff
@@ -14,108 +14,213 @@ See the License for the specific language governing permissions and
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Total CPU time</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>CPU time vtgate</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>CPU time vttablet</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Total Allocs bytes</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Allocs bytes vtgate</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Allocs bytes vttablet</span>
+        </div>
+
+        <div className="macrobench__component__container flex">
+          <Swiper
+            effect={"cards"}
+            grabCursor={true}
+            modules={[EffectCards]}
+            onSlideChange={handleSlideChange}
+            initialSlide={currentSlideIndexMobile}
+            className="mySwiper"
+          >
+            <SwiperSlide>
+              <div className="macrobench__data flex--column">
+                <h4>Impoved by %</h4>
+                <span>{fixed(data.diff.Diff.qps.total, 0)}</span>
+                <span>{fixed(data.diff.Diff.qps.reads, 0)}</span>
+                <span>{fixed(data.diff.Diff.qps.writes, 0)}</span>
+                <span>{fixed(data.diff.Diff.qps.other, 0)}</span>
+                <span>{fixed(data.diff.Diff.tps, 0)}</span>
+                <span>{fixed(data.diff.Diff.latency, 0)}</span>
+                <span>{fixed(data.diff.Diff.errors, 0)}</span>
+                <span>{fixed(data.diff.Diff.reconnects, 0)}</span>
+                <span>{fixed(data.diff.Diff.time, 0)}</span>
+                <span>{fixed(data.diff.Diff.threads, 0)}</span>
+                <span>
+                  {fixed(data.diff.DiffMetrics.TotalComponentsCPUTime, 0)}
+                </span>
+                <span>
+                  {fixed(data.diff.DiffMetrics.ComponentsCPUTime.vtgate, 0)}
+                </span>
+                <span>
+                  {fixed(data.diff.DiffMetrics.ComponentsCPUTime.vttablet, 0)}
+                </span>
+                <span>
+                  {formatByteForGB(
+                    data.diff.DiffMetrics.TotalComponentsMemStatsAllocBytes
+                  )}
+                </span>
+                <span>
+                  {fixed(
+                    data.diff.DiffMetrics.ComponentsMemStatsAllocBytes.vtgate,
+                    0
+                  )}
+                </span>
+                <span id="borderNone">
+                  {fixed(
+                    data.diff.DiffMetrics.ComponentsMemStatsAllocBytes.vttablet,
+                    0
+                  )}
+                </span>
+              </div>
+            </SwiperSlide>
+            <SwiperSlide>
+              <div className="macrobench__data flex--column">
+                <h4>{gitRefLeft ? gitRefLeft : "Left"}</h4>
```

Comment:
We need to trim the length of the commit SHAs we are rendering otherwise it overflows as shown in the screenshot below.

![image](https://github.com/vitessio/arewefastyet/assets/35779988/7f655fab-4ab5-44f9-8673-f67c3c372a33)